scripts/migrate-chart-value/main.go

```diff
@@ -0,0 +1,213 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"io"
+	"log"
+	"os"
+
+	"gopkg.in/yaml.v2"
+)
+
+var (
+	inputFilePath      string
+	targetChartVersion string
+	outputFilePath     string
+)
+
+const (
+	NEW_VERSION = "v1.8.0"
+	OPERATOR    = "operator"
+	STARROCKS   = "starrocks"
+)
+
+var _starrocksKeys = []string{"nameOverride", "initPassword", "timeZone", "datadog", "starrocksCluster",
+	"starrocksFESpec", "starrocksCnSpec", "starrocksBeSpec", "secrets", "configMaps", "feProxy"}
+
+var _operatorKeys = []string{"global", "timeZone", "nameOverride", "starrocksOperator"}
+
+func main() {
+	flag.Usage = func() {
+		fmt.Fprintf(os.Stderr, `
+This tool is used to upgrade or downgrade the version of values.yaml for kube-starrocks chart.
+
+If the chart version is less than v1.8.0, the values.yaml is in the following format:
+key1: value1
+key2: value2
+
+If the chart version is greater than or equal to v1.8.0, the values.yaml will be changed to the following format:
+operator:
+  aaa: bbb
+starrocks:
+  xxx: yyy
```

Comment:
you have to change these lines accordingly.

Reply:
sorry, I forget it